spring-boot-project/spring-boot-dependencies/pom.xml

```diff
@@ -113,7 +113,7 @@
 		<logback.version>1.2.3</logback.version>
 		<lombok.version>1.16.18</lombok.version>
 		<mariadb.version>2.1.2</mariadb.version>
-		<micrometer.version>1.0.0-rc.1</micrometer.version>
+		<micrometer.version>1.0.0-SNAPSHOT</micrometer.version>
```

Comment:
If you are satisfied with what's been done in the rest of this PR, I can go ahead and release 1.0.0-rc.2